app.js

```diff
@@ -0,0 +1,764 @@
+const { App } = require("@slack/bolt");
+const dotenv = require("dotenv");
+const axios = require("axios");
+const fs = require("fs");
+
+// import consts
+const { new_user } = require("./settings/new_user");
+
+// import modal blocks
+const modals = require("./blocks/modals");
+const {
+  launch_modal,
+  approver_details,
+  quote_line_details,
+  deal_stats,
+} = modals;
+
+// import message blocks
+const {
+  app_home,
+  channel_exists,
+  channel_created,
+  discount_approved,
+  discount_mention,
+  channel_message,
+  redirect_home,
+  thread_approved,
+  thread_ask,
+  thread_error,
+} = require("./blocks/messages");
+
+// import database functions
+const {
+  storeTokens,
+  storeLocalInstallation,
+  retrieveTokens,
+  fetchLocalInstallation,
+} = require("./db/helpers");
+
+// initialize env variables
+dotenv.config();
+
+// intialize app
+const app = new App({
+  signingSecret: process.env.SLACK_SIGNING_SECRET,
+  clientId: process.env.SLACK_CLIENT_ID,
+  clientSecret: process.env.SLACK_CLIENT_SECRET,
+  stateSecret: process.env.SLACK_STATE_SECRET,
+  scopes: [
+    "channels:history",
+    "channels:manage",
+    "chat:write",
+    "chat:write.public",
+    "commands",
+    "users:read",
+    "channels:read",
+    "links:read",
+  ],
+  installerOptions: {
+    userScopes: ["channels:history"],
+  },
+  installationStore: {
+    storeInstallation: async (installation) => {
+      await storeTokens(installation);
+      return storeLocalInstallation(installation);
+    },
+    fetchInstallation: async ({ teamId }) => {
+      const tokens = await retrieveTokens(teamId);
+      const installation = fetchLocalInstallation(tokens, teamId);
+      return installation;
+    },
+  },
+});
+
+/* UTILITY FUNCTIONS 🧰 */
+
+// check if approver users exist at each level
+const is_approvers_configured = (approver_users) => {
```

Comment:
Suggestion: Make function names consistently camelCase. Checked with Malika, and our team follows the Google Style Guide, generally, here's their [entry on functions](https://google.github.io/styleguide/jsguide.html#features-functions)